proposed/event-dispatcher-meta.md

```diff
@@ -37,85 +37,84 @@ The Working Group identified four possible workflows for event passing, based on
+However, Stoppable Events (the alternative chain case) also needed to have a channel by which to indicate that further listeners should not be called.  That could be done either by:
 
-* Modifying the Task (`stopPropagation()`)
+* Modifying the Event (e.g., calling a `stopPropagation()` method)
 * Returning a sentinel value from the listener (`true` or `false`) to indicate that propagation should terminate.
-* Evolving the Task to be stopped (`withPropagationStopped()`)
+* Evolving the Event to be stopped (`withPropagationStopped()`)
 
-Of those, the first would mandate a mutable Task in at least some cases.  The second would mandate a mutable Task as the return value was already in use.  And the third seemed unnecessarily ceremonial and pedantic.
+Of those, the first would mandate a mutable Event in at least some cases.  The second would mandate a mutable Event as the return value was already in use.  And the third seemed unnecessarily ceremonial and pedantic.
 
-Having listeners return evolvable tasks also posed a challenge.  That pattern is not used by any known implementations in PHP or elsewhere.  It also relies on the listener to remember to return the object (extra work for the listener author) and to not return some other, new object that might not be fully compatible with later listeners (such as a subclass or superclass of the event).
+Having listeners return evolvable events also posed a challenge.  That pattern is not used by any known implementations in PHP or elsewhere.  It also relies on the listener to remember to return the Event (extra work for the listener author) and to not return some other, new object that might not be fully compatible with later listeners (such as a subclass or superclass of the Event).
 
 Immutable events also rely on the event definer to respect the admonition to be immutable.  Events are, by nature, very loosely designed and the potential for implementers to ignore that part of the spec, even inadvertently, is high.
 
 That left two possible options:
 
-* Allow tasks to be mutable.
+* Allow events to be mutable.
 * Require, but be unable to enforce, immutable objects with a high-ceremony interface, more work for listener authors, and a higher potential for breakage that may not be detectable at compile time.
 
-Given those options the Working Group felt mutable tasks were the safer alternative.
+Given those options the Working Group felt mutable events were the safer alternative.
 
-The Message Notification use case, however, has no need for return communication so those issues are moot.  Those objects therefore MUST be treated as immutable, regardless of what the object's methods are.
+That said, there is no requirement that an Event be mutable.  Implementers should provide mutator methods on an Event object if and only if it is necessary and appropriate to the use case at hand.
```

Comment:
I would make this paragraph bold, as I think it needs emphasis. There is a vocal contingent of evaluators who feel immutability is required; we can point them to this section to note that immutability is not _against the specification_, but simply _not required_.

Reply:
"That said, there is no requirement that an Event be mutable" => maybe I'm just non-native, but shouldn't we say "that an Event may be mutable"?